requirements.txt

```diff
@@ -19,3 +19,4 @@ pydantic==2.9.1
 pyyaml
 humanfriendly
 responses
+gitpython
```

Comment:
Is this used somewhere, or added by accident?

Reply:
I was getting this when trying to run it before.
```
$ python3 python/terra_summary_statistics.py -b anvil-datastorage -w AnVIL_SSI_Demo_Staging -d ~/Desktop/data_dictionary_base.tsv
Traceback (most recent call last):
  File "/Users/samn/ops-terra-utils/python/terra_summary_statistics.py", line 10, in <module>
    from utils.request_util import RunRequest
  File "/Users/samn/ops-terra-utils/python/utils/request_util.py", line 2, in <module>
    from python.tests.utils.mock_util import mock_responses
  File "/Users/samn/ops-terra-utils/python/tests/utils/mock_util.py", line 7, in <module>
    import git
ModuleNotFoundError: No module named 'git'
```